src/dotnet/commands/dotnet-new/FSharp_Sln/$projectName$.sln

```diff
@@ -0,0 +1,11 @@
+Microsoft Visual Studio Solution File, Format Version 15.00
```

Comment:
We don't have support for that currently in `dotnet new` so I just used the existing infrastructure. I can endevor to make this support, if you think it is OK. :)